data/app.drey.PaperPlane.metainfo.xml.in.in

```diff
@@ -35,18 +35,25 @@
     <control>touch</control>
   </recommends>
   <requires>
-    <display_length compare="ge">360</display_length>
+    <display_length compare="ge">small</display_length>
```

Comment:
Those named sizes should not be used according to [this issue](https://github.com/ximion/appstream/issues/335). The app is tested to work on devices with a minimum width of 360 pixels, so this line was correct.

Reply:
I followed this [https://tecnocode.co.uk/2021/07/12/add-metadata-to-your-app-to-say-what-inputs-and-display-sizes-it-supports/](url) but I got it. It should be small then.

Reply:
![image](https://github.com/paper-plane-developers/paper-plane/assets/64592232/e7b7515c-fa20-46e6-b6ff-2d9216e920c7)

This is a much better and more updated article [https://blogs.gnome.org/tbernard/2021/09/07/ready-for-software-41/#device-support](url)

Reply:
So, this needs to be reverted.